modules/platform/nuxeo-platform-usermanager/src/test/java/org/nuxeo/ecm/platform/usermanager/TestUserManager.java

```diff
@@ -1440,18 +1452,19 @@ protected void doTestGetQueryForPattern(boolean compat) throws Exception {
 
         // basic query, directory configured for "subinitial" substring match type
         query = getQueryForPattern("foo");
-        assertEquals("((firstName ILIKE 'foo%') OR (lastName ILIKE 'foo%') OR (username ILIKE 'foo%'))",
-                query);
+        assertEquals("((firstName ILIKE 'foo%') OR (lastName ILIKE 'foo%') OR (username ILIKE 'foo%'))", query);
```

Comment:
Reminder for next time: please avoid reformatting code in the same commit as other regular changes, it slows down review.

Reply:
I'll keep it in mind for next time, thanks for the reminder